doc/rados/operations/pools.rst

```diff
@@ -276,21 +276,21 @@ You may set values for the following keys:
 
 ``compression_algorithm``
 
-:Description: Sets inline compression algorithm to use for underlying BlueStore. This setting overrides the `global setting <http://docs.ceph.com/docs/master/rados/configuration/bluestore-config-ref/#inline-compression>`_ of ``bluestore compression algorithm``.
+:Description: Sets inline compression algorithm to use for underlying BlueStore. This setting overrides the `global setting <https://docs.ceph.com/en/latest/rados/configuration/bluestore-config-ref/#inline-compression>`_ of ``bluestore compression algorithm``.
```

Comment:
```
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/rados/operations/pools.rst:3: WARNING: Duplicate explicit target name: "global setting".
/home/jenkins-build/build/workspace/ceph-pr-docs/doc/rados/operations/pools.rst:3: WARNING: Duplicate explicit target name: "global setting".
```

see https://jenkins.ceph.com/job/ceph-pr-docs/36648/console

there are two options:

- unnamed link, as suggested by https://stackoverflow.com/questions/5464627/how-to-have-same-text-in-two-links-with-restructured-text
- add a named hyper link: see https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks